### PR TITLE
docs: ui.toggle_button

### LINKED ISSUE #838: docs: ui.toggle_button

### PR DESCRIPTION
**Pull request by AkshatJawne**
Closes #838

### CODE REVIEW

File: plugins/ui/docs/components/toggle_button.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+```
+
+## UI Recommendations
+
+Consider using an [`checkbox`](./checkbox.md) if wanting to represent a binary choice for the user.
+
+
+## Content
+
+A toggle button accepts a label, an icon, or both as children
```
Comment (margaretkennedy) [inline plugins/ui/docs/components/toggle_button.md:20]:
```suggestion
A toggle button accepts a label, an icon, or both as children.
```

File: plugins/ui/docs/components/toggle_button.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+from deephaven import ui
+
+
+@ui.component
+def ui_toggle_button_value_examples():
+    selected, set_selected = ui.use_state(False)
+    return ui.flex(
+        ui.heading("Toggle Button (uncontrolled)"),
+        ui.toggle_button("Pin", default_selected=True, width="90px"),
+        ui.heading("Toggle Button (controlled)"),
```
Comment (dsmmcken) [inline plugins/ui/docs/components/toggle_button.md:58]:
```suggestion
        ui.text("Toggle Button (uncontrolled)"),
        ui.toggle_button("Pin", default_selected=True, width="90px"),
        ui.text("Toggle Button (controlled)"),
```

shhh.

File: plugins/ui/docs/components/toggle_button.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+A toggle button is not selected by default. Use the `default_selected` prop to set the initial state (uncontrolled) or the `is_selected` prop to control the selected state.
+
+```python
+from deephaven import ui
+
+
+@ui.component
+def ui_toggle_button_value_examples():
+    selected, set_selected = ui.use_state(False)
+    return ui.flex(
```
Comment (dsmmcken) [inline plugins/ui/docs/components/toggle_button.md:55]:
```suggestion
    return [
```

it's implicitly wrapped in a flex column, no need to add one.

File: plugins/ui/docs/components/toggle_button.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+        ui.heading("Toggle Button (uncontrolled)"),
+        ui.toggle_button("Pin", default_selected=True, width="90px"),
+        ui.heading("Toggle Button (controlled)"),
+        ui.toggle_button(
+            "Pin", is_selected=selected, on_change=set_selected, width="90px"
+        ),
+        direction="column",
+    )
```
Comment (dsmmcken) [inline plugins/ui/docs/components/toggle_button.md:63]:
```suggestion
    ]
```

File: plugins/ui/docs/components/toggle_button.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+my_toggle_button_is_emphasized_example = ui.toggle_button(
+    "Pin",
+    is_emphasized=True,
+)
+```
+
+
+## Static Color
+
+The `static_color` prop can be used when the toggle button is placed over a color background.
```
Comment (dsmmcken) [inline plugins/ui/docs/components/toggle_button.md:144]:
```suggestion
The `static_color` prop can be used when the toggle button is placed over a colored background.
```

**Review by dsmmcken — CHANGES_REQUESTED** [review]
Minor, but otherwise good.

**Review by dsmmcken — APPROVED** [review]

File: plugins/ui/docs/components/toggle_button.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+from deephaven import ui
+
+
+my_toggle_button = ui.toggle_button(ui.icon("pin"), ui.text("Pin content"))
+```
+
+
+## Accessibility
+
+If there is no text that is passed into the toggle button, and hence, it has no visible label, the `aria_label` prop should be set for accessibility.
```
Comment (margaretkennedy) [inline plugins/ui/docs/components/toggle_button.md:32]:
```suggestion
If no text is passed into the toggle button, and hence, it has no visible label, the `aria_label` prop should be set for accessibility.
```

File: plugins/ui/docs/components/toggle_button.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+    ]
+
+
+my_toggle_button_value_examples = ui_toggle_button_value_examples()
+```
+
+
+## Events
+
+The `on_change` property is triggered whenever the value in the toggle button group selection is changed.
```
Comment (margaretkennedy) [inline plugins/ui/docs/components/toggle_button.md:71]:
```suggestion
The `on_change` property is triggered whenever the toggle button group selection value is changed.
```

**Review by dsmmcken — APPROVED** [review]

**Pull request merged**